src/Compilers/CSharp/Portable/Symbols/Metadata/PE/SymbolFactory.cs

```diff
@@ -57,6 +57,11 @@ internal override bool IsVolatileModifierType(PEModuleSymbol moduleSymbol, TypeS
             return type.SpecialType == SpecialType.System_Runtime_CompilerServices_IsVolatile;
         }
 
+        internal override bool IsConstModifierType(PEModuleSymbol moduleSymbol, TypeSymbol type)
+        {
+            return type.ToDisplayString() == WellKnownTypes.GetMetadataName(WellKnownType.System_Runtime_CompilerServices_IsConst);
+        }
```

Comment:
This seems simple enough to fix now vs. needing a PROTOTYPE comment to fix later.